changelogs/fragments/704-doas-set-correct-default-values.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+- doas - The module expects become_user and become_flags to by empty string by default.
```

```suggestion
- doas become plugin - the plugin expects become_user and become_flags to by empty strings by default (https://github.com/ansible-collections/community.general/pull/704).
```
I'm not really sure what this message tells me as a user though.

---

"It was broken, not it works!". But indeed, I can rephrase that a bit.